stacks/dhis2-db/seed.sh

```diff
@@ -23,11 +19,17 @@ if [[ -n $DATABASE_ID ]]; then
   DATABASE_MANAGER_ABSOLUTE_URL="$DATABASE_MANAGER_URL/databases/$DATABASE_ID/download"
   echo "DATABASE_MANAGER_ABSOLUTE_URL: $DATABASE_MANAGER_ABSOLUTE_URL"
 
+  exec_psql "create extension if not exists postgis"
+  exec_psql "create extension if not exists pg_trgm"
+  exec_psql "create extension if not exists btree_gin"
+
   tmp_file=$(mktemp)
-  curl --fail -L "$DATABASE_MANAGER_ABSOLUTE_URL" -H "Authorization: $IM_ACCESS_TOKEN" > "$tmp_file"
+  curl --connect-timeout 10 --retry 5 --retry-delay 1 --fail -L "$DATABASE_MANAGER_ABSOLUTE_URL" -H "Authorization: $IM_ACCESS_TOKEN" > "$tmp_file"
 # Try pg_restore... Or gzipped sql
-  (pg_restore -U postgres -d "$DATABASE_NAME" -j 4 "$tmp_file") || \
-  (gunzip -v -c "$tmp_file" | psql -U postgres -d "$DATABASE_NAME")
+# pg_restore often returns a non zero return code due to benign errors resulting in executing of gunzip despite the restore being successful
+# gunzip will fail because the input isn't gzipped causing the whole seed script to fail... Which is why there's a "|| true" at the end
+  (pg_restore --verbose --create -U postgres -d "$DATABASE_NAME" -j 4 "$tmp_file") || \
+  (gunzip -v -c "$tmp_file" | psql -U postgres -d "$DATABASE_NAME") || true
```

Comment:
If we can't avoid these "benign errors" from `pg_restore`, then `|| true` is fine I suppose. 🤔

Reply:
The current dumps contain them. I would like to have "better" behaving dumps but... @Philip-Larsen-Donnelly ?